jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -114,6 +116,8 @@
             [
               "openstack-horizon-plugin-neutron-fwaas-ui",
               "openstack-horizon-plugin-neutron-vpnaas-ui",
+              "python-vmware-nsxlib",
+              "python-vmware-nsx",
```

Comment:
I think that this will track python-vmware-nsx* in all branches, except master and ocata. Makes sense for ocata, but maybe you want to track also master?

Reply:
there are no python-vmware-nsx and python-vmware-nsxlib packages in C:O:Master yet, so the job will fail, but I guess I could add them to the job now and correct the C:O:M situation after